data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditMessage.java

```diff
@@ -48,14 +50,25 @@
 
     protected static final ObjectMapper MAPPER = new ObjectMapper();
 
+    @JsonProperty("leafResourceId")
+    private final String leafResourceId;  //leafResource Id requested
+
+    @JsonProperty("token")
+    private final String token;  //token for this request
+
     @JsonProperty("userId")
-    protected final String userId; //Unique identifier for the user.
+    @Nullable
+    protected final String userId; // Unique identifier for the user.
 
     @JsonProperty("resourceId")
-    protected final String resourceId;  //Resource that that is being asked to access.
+    @Nullable
+    protected final
+    String resourceId;  //Resource that that is being asked to access.
```

Comment:
Is there a reason this has been split over 2 lines?
```suggestion
    protected final String resourceId;  //Resource that that is being asked to access.
```